### ISSUE TITLE
Not working in gradle because of ${revision}

### ISSUE THREAD

**Issue by pratrajesh** [semver4j/semver4j#67]
Working fine in maven project but getting below error for gradle project while building.

```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileJava'.
> Could not resolve all files for configuration ':compileClasspath'.
   > Could not resolve org.semver4j:semver4j:2.0.1.
     Required by:
         project :
      > Could not resolve org.semver4j:semver4j:2.0.1.
         > inconsistent module metadata found. Descriptor: org.semver4j:semver4j:${revision} Errors: bad version: expected='2.0.1' found='${revision}'
      > Could not resolve org.semver4j:semver4j:2.0.1.
         > inconsistent module metadata found. Descriptor: org.semver4j:semver4j:${revision} Errors: bad version: expected='2.0.1' found='${revision}'
```

**Following change may resolve this error, tested in my local.**
- In pom.xml change ${revision} to particular version i.e 2.0.1

ex : 
```
    <groupId>org.semver4j</groupId>
    <artifactId>semver4j</artifactId>
    <version>2.0.1</version>
    <packaging>jar</packaging>
```

**Comment by piotrooo** [comment]
Thanks. I'll investigate it.

**Comment by piotrooo** [comment]
Fixed in 2.1.0
